### DISCUSSION TITLE
Event series does not show when unavailable anymore

### DISCUSSION THREAD

**Discussion by holgerm** [pretix/pretix#4379] [Q&A]
I have defined an event series with three sub-events at different dates. I defined a terminal date for both the event itself as well for each date / product within. But on the customer page the clear hint saying that this event (series) is not available anymore is not shown. I typically see this hint on single events when they are past due date. Am I missing something, or is this intentionally? 

I would like to see that hint, since it is much easier for our customers than scrolling back and forth through the calendar just to learn that there is no available date anymore.

**Comment by raphaelm** [comment]
Can you add screenshots? I don't follow which hints you are talking about exactly.

**Comment by holgerm** [comment]
<img width="1149" alt="Screenshot 2024-08-05 at 10 57 42" src="https://github.com/user-attachments/assets/5b619cb3-25cd-44b5-8298-c829e1b77570">

This is what I see on single events only but not for my series.
